jep/202/README.adoc

```diff
@@ -186,6 +196,26 @@ This temporary role creation does not exist in all clouds nor other artifact rep
 
 We opted for the first, simpler option.
 
+== Backwards Compatibility
```

Comment:
eek, GitHub diff is getting confused, may trigger merge conflicts with #98 but OK